### ISSUE TITLE
[Feature request] More options to control padding

### ISSUE THREAD

**Issue by andrii-bodnar** [delucis/astro-og-canvas#69]
Hi, thanks for the great package!

I am using the `bgImage` option and the background contains some visuals on the right side of the image. e.g. 1/3 of the background. In case the page title and description are quite long, they **overlap** the visuals in the background. 

I couldn't adjust the `padding` in this case because it needs about 200 pixels of right padding and in the case of using this value, the texts will appear with unwanted top and left padding with this value.

It would be great to have a way to control each padding (top, left, right, bottom) separately instead of using the same value for all of them.

**Comment by delucis** [comment]
Agreed! I unfortunately haven’t had much time to spend on this but would very much like to revisit some of these limited APIs to make them more flexible.
